client/lib/user/user.d.ts

```diff
@@ -70,9 +70,11 @@ export type OptionalUserData = {
 	primary_blog_is_jetpack: boolean;
 	primary_blog_url: string;
 	site_count: number;
+	jetpack_site_count?: number;
```

Comment:
Will use these fields in client/jetpack-cloud/controller.ts#noSite in a later PR.